.github/workflows/checks.yml

```
@@ -38,10 +38,12 @@ jobs:
           output: report-markdown
           annotations: failed-tests
           prnumber: ${{ steps.findPr.outputs.number }}
+      - name: Run black
+        run: poetry run black .
+      - name: Check imports alphabetized
+        run: poetry run isort ./app ./tests
```

Same note as the change in the API - we can't do this within GitHub itself during the CI/CD actions, but we can make the change for the local development workflow.  This way we ensure the changes are captured and reviewed just like everything else!

```suggestion
      - name: Check imports alphabetized
        run: poetry run isort --check-only ./app ./tests
```